python/knowledges/batch_upload/upload_missing_files.py

```
@@ -0,0 +1,325 @@
+    if missing_files:
+        await uploader.upload_missing_files(missing_files)
+        
+        # Show summary
+        print("\n" + "=" * 60)
+        print("Upload Summary:")
+        print("=" * 60)
+        print(f"✅ Successfully uploaded: {len(uploader.successful_uploads)} files")
+        print(f"❌ Failed uploads: {len(uploader.failed_uploads)} files")
+        
+        if uploader.failed_uploads:
+            print(f"\nUpload failures:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)